api/shas.go

```diff
@@ -79,6 +79,7 @@ func (h SHAsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 		seen := mapset.NewSet()
 		for _, testRun := range testRuns.AllRuns() {
+			// nolint:staticcheck // TODO: Fix staticcheck lint error.
```

Comment:
Is this related to https://github.com/web-platform-tests/wpt.fyi/pull/3324/files#r1199003356

Reply:
Yep you're correct!